scripts/Update-Proto.ps1

```diff
@@ -0,0 +1,49 @@
+$ErrorActionPreference = 'Stop'
+$VerbosePreference = 'Continue'
+
+try {
+    $root = Split-Path -Path $PSScriptRoot -Parent
+    Push-Location $root
+
+    @('go', 'protoc') | ForEach-Object {
+        if ( $null -eq (Get-Command $_ -ErrorAction Ignore) ) {
+            throw "executable $_ not in path"
+        }
+    }
+
+    # Install protobuild and co; rely on tools.go to vendor correct dependencies
```

Comment:
How does the tool.go file work? Is this just so that the versions vendored by the project match the ones used here?